.travis.yml

```diff
@@ -75,6 +77,13 @@ before_install:
     - make -j2 install > /dev/null
     - ls -la /tmp/gsl-2.0/lib
     - cd ..
+    - wget ftp://ftp.gnu.org/gnu/gsl/gsl-latest.tar.gz
+    - tar zxpf gsl-latest.tar.gz
+    - cd gsl-*
```

Comment:
This actually cd's into gsl-1.15 and then uses the prefix of /tmp/gsl-latest , which is wrong. I wasn't clear when I said "latest". What I meant is the master branch of GSL. So please change it to git clone GSL and build the master branch.